src/jail_run.c

```diff
@@ -73,6 +73,14 @@ static const char FIREJAIL_FINISH_OPT[] = "--";
 
 static const char PRIVILEGED_GROUP[] = "privileged";
 
+static gboolean jail_run_trace_enabled = FALSE;
+
+void
+jail_run_enable_trace(void)
+{
+    jail_run_trace_enabled = TRUE;
+}
```

Comment:
If you insist on passing parameters via global variables, then expose this one as a variable. The function is unnecessary.